docs/csharp/tutorials/console-webapiclient.md

```diff
@@ -261,6 +261,12 @@ the following changes to the declaration of the `name` field in repo.cs:
 public string Name { get; set; }
 ```
 
+To use `[JsonPropertyName]` attribute, you will need to add the <xref:System.Text.Json.Serialization> namespace to the `using` statements:
```

Comment:
I think "using statement" usually refers to the statement that disposes a resource, while "using directive" refers to importing namespaces.

- https://docs.microsoft.com/dotnet/csharp/language-reference/keywords/using-directive
- https://docs.microsoft.com/dotnet/csharp/language-reference/keywords/using-statement

Reply:
As I've copy and pasted the sentence from elsewhere in the tutorial, it might be better to correct it everywhere.